src/interface/src/app/treatments/map-stands/map-stands.component.ts

```diff
@@ -174,7 +179,7 @@ export class MapStandsComponent implements OnChanges, OnInit {
   }
 
   private generateFillColors(stands: TreatedStand[]) {
-    const defaultColor = '#00000050';
+    const defaultColor = '#F6F6F650';
```

Comment:
I don't know if there's a more central place to define some of these hex values on the TS side, but it might be easier to maintain constants.

Reply:
yeah good point, its starting to get a bit out of hand. I wasn't sure how to group this in a way that wasn't a pain to use ultimately on the templates, but gonna take a look!

Reply:
@lastminutediorama made some changes for this, let me know what you think.